style.css

```diff
@@ -0,0 +1,80 @@
+body {
+  background: white;
+}
+
+#navbar {
+  display: flex;
+  justify-content: flex-end;
+  position: fixed;
+  top: 0;

```

Comment:
- Please work on your CSS as it is not structuring your webpage properly, PLease try fixing your CSS, The image below is what I get when I open your webpage

You can reach out to the code reviewer on slack `@kanu mike chibundu` if you find this feedback confusing.
##### Your design 
![Screenshot from 2022-07-01 20-17-48](https://user-images.githubusercontent.com/58771507/176958319-90843182-6b17-45ab-bbdf-fddbc9caf64a.png)